peerconnection.go

```diff
@@ -1916,24 +1910,22 @@ func (pc *PeerConnection) GetStats() StatsReport {
 		pc.iceTransport.collectStats(statsCollector)
 	}
 
-	if pc.sctpTransport != nil {
```

Comment:
`sctpTransport` is never nil. If something is depending on this behavior I would like to know